migrations/2024-06-10-084722_create_org_authentication_table/down.sql

```diff
@@ -0,0 +1,5 @@
+-- This file should undo anything in `up.sql`
```

Comment:
Do we need to remove `org_auth_method_unique` constraint as well? Or will it be automatically removed?

Reply:
dropping the org_authentication_methods table will also remove the org_auth_method_unique constraint, so we typically don't need to explicitly drop the constraint